src/hbs.rs

```diff
@@ -6,12 +6,12 @@ use hpos_hc_connect::{hpos_agent::get_hpos_config, CoreAppAgent};
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Deserialize)]
-struct HostingCriteria {
+pub struct HostingCriteria {
     #[allow(dead_code)]
-    id: String,
+    pub id: String,
     #[allow(dead_code)]
-    jurisdiction: String,
-    kyc: KycLevel,
+    pub jurisdiction: String,
```

Comment:
let's have jurisdiction and id be of type `Option<String>` so that the default can be `None` to keep the code clean